pkg/cmd/get/get_build_logs_test.go

```diff
@@ -63,6 +63,32 @@ func TestGetTektonLogsForRunningBuild(t *testing.T) {
 	}
 }
 
+func TestGetTektonLogsForRunningBuildWithPendingPod(t *testing.T) {
+	commonOpts := opts.NewCommonOptionsWithFactory(fake.NewFakeFactory())
+	commonOpts.BatchMode = true
+	testCaseDir := path.Join("test_data", "get_build_logs", "tekton_build_logs_pending")
+
+	activities := tekton_helpers_test.AssertLoadSinglePipelineActivity(t, testCaseDir)
+	jxClient := jxfake.NewSimpleClientset(activities)
+
+	tektonObjects := []runtime.Object{tekton_helpers_test.AssertLoadSinglePipelineRun(t, testCaseDir)}
```

Comment:
Same as above

Reply:
OOI, would it be easier to write these types of tests as integration tests? I think there is nothing wrong with integration tests, they also run fast and you would have considerably less mocking going on. WDYT?